### ISSUE TITLE
can't build with 7.10.2 due to dependency on bitset

### ISSUE THREAD

**Issue by miguel-negrao** [kaoskorobase/hsc3-server#13]
https://github.com/lambda-llama/bitset doesn't build with ghc7.10.2, the integer GMP library has been completely changed apparently and bitset hasn't been updated since 2014. Do you think bitset will eventually be fixed to work with ghc7.10.2 ?

**Comment by kaoskorobase** [comment]
I don't know, have you tried to contact the authors?

**Comment by miguel-negrao** [comment]
> I don't know, have you tried to contact the authors?

No, but there is an issue since June that as gone without reply...

https://github.com/lambda-llama/bitset/issues/17
